sublime/User/Preferences.sublime-settings

```diff
@@ -19,8 +19,15 @@
 		"*.sassc"
 	],
 	"bold_folder_labels": true,
-	"color_scheme": "Packages/User/SublimeLinter/Material Spacegray (SL).tmTheme",
+	"color_scheme": "Packages/Edge Theme/Edge Night Sky.tmTheme",
```

Comment:
AHAH! Isn't this one too much opinionated? 😂 I'm ready to accept it since i cannot verify myself how much cool it is 😉